redbeams-api/src/main/java/com/sequenceiq/redbeams/validation/DatabaseVendorAndServiceValidator.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.redbeams.validation;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.DatabaseVendor;
+import com.sequenceiq.cloudbreak.validation.ValidatorUtil;
+import com.sequenceiq.cloudbreak.validation.externaldatabase.SupportedDatabaseEntry;
```

Comment:
These are imported from core-api. I think it would be better not to use things from there, as core-api belongs to core. We should use things from common if it is to be shared.

Reply:
I agree with you, but I don't want to move SupportedDatabaseEntry et al. at this time, since I think this PR is big enough. It should be done later.